CHANGELOG.rst

```diff
@@ -22,6 +22,25 @@ People with a "+" by their names contributed a patch for the first time.
 * Joris Van den Bossche
 
 
+Version 0.10.1 (2021-07-??)
```

Comment:
@caspervdw any preference for releasing this as 0.10.1 or 0.11? 
We didn't yet do micro releases before, but since this is only a tiny bugfix release I thought it was fitting here. For me both is fine.

Reply:
I think we had a 0.7.1 as well. I’d opt for a micro release.